src/interfaces/store-interfaces/state-interfaces/restoration-state-interface.ts

```diff
@@ -11,6 +11,8 @@ export interface RestorationStateIF {
   businessNameValid: boolean
   type: RestorationTypes
   expiry?: string // YYYY-MM-DD
+  relationships?: RelationshipTypes[]
+  isRestorationTypeValid: boolean
```

Comment:
Sorry, I just noticed this now, but how about renaming this flag to be consistent with the other validity flags in this object?